.changeset/poor-mice-allow.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fuel-ts/contract": patch
+"@fuel-ts/account": patch
+"@fuel-ts/program": patch
+---
+
+chore: upgrading `@fuels/vm-asm` to `0.57.1`
```

Comment:
> [!WARNING]
> Blocked by:
> - https://github.com/FuelLabs/fuel-vm/pull/835

CI checks for this PR will only pass when the above is merged and released.

Expected version for the next release: `0.57.1`